pythclient/calendar.py

```diff
@@ -143,20 +176,35 @@ def get_next_market_close(asset_type: str, dt: datetime.datetime) -> int:
                     microsecond=0,
                 )
                 next_market_close += datetime.timedelta(days=1)
-        elif dt.date() in EQUITY_HOLIDAYS:
-            next_market_open = get_next_market_open(
-                asset_type, dt + datetime.timedelta(days=1)
-            )
-            next_market_close = (
+        elif dt.date() in NYSE_HOLIDAYS:
+            next_market_open = get_next_market_open(asset_type, dt)
```

Comment:
fixed this because there was a bug where this would return incorrect answer if the given date is a holiday and the next day is early close holiday